frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts

```diff
@@ -33,12 +33,9 @@ import {ChangeDetectorRef, Component, Input, OnInit} from '@angular/core';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {Highlighting} from "core-components/wp-fast-table/builders/highlighting/highlighting.functions";
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
-import {WorkPackageCacheService} from "core-components/work-packages/work-package-cache.service";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
-import {concatAll} from "rxjs/operators";
-import {combineLatest} from "rxjs";
 import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
-import {ISchemaProxy, SchemaProxy} from "core-app/modules/hal/schemas/schema-proxy";
+import {ISchemaProxy} from "core-app/modules/hal/schemas/schema-proxy";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e71869ec897476f98b23dceb4c3c60722bb3632a/frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts#L39"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>